packages/fx-core/src/component/driver/aad/create.ts

```diff
@@ -321,6 +332,28 @@ export class CreateAadAppDriver implements StepDriver {
       aadAppState.clientSecret = aadAppSecret.value.result;
       aadAppState.objectId = aadAppObjectId.value.result;
       const outputs = mapStateToEnv(aadAppState, outputEnvVarNames);
+
+      await envUtil.writeEnv(context.projectPath, env, envUtil.map2object(outputs));
+
+      // Remind user to manually remove aadApp/update action from teamsapp.yml file, or they will fail at aadApp/update action
+      const teamsappYamlPath = pathUtils.getYmlFilePath(context.projectPath, env);
+      const yamlProjectModel = await metadataUtil.parse(teamsappYamlPath, env);
+      if (yamlProjectModel.isErr()) {
+        return err(yamlProjectModel.error);
+      }
+      const teamsAppUpdate = yamlProjectModel.value.provision?.driverDefs.find(
+        (d: any) => d.uses === "aadApp/update"
+      );
+      if (teamsAppUpdate) {
+        const message = getLocalizedString(
+          logMessageKeys.needManualRemoveAadUpdate,
+          teamsappYamlPath
+        );
+        await context.ui!.showMessage("warn", message, true);
```

Comment:
should we try to update the manifest first before show the warning? user may already has permission to update the app

Reply:
By the way, have we reached the UX team to review the experience? We may reach them for feedback if haven't. Asking users to remove a step directly seems a little weird. We probably need to tell them why they need to remove that step and what they need to do before remove.

Reply:
> should we try to update the manifest first before show the warning? user may already has permission to update the app

We go into this function, because create API respond 403 error with insufficient permission. Which means user don't have permission to update as well.